src/app/Http/Controllers/API/PollController.php

```diff
@@ -50,7 +62,7 @@ public function store(Request $request, Presentation $presentation)
      */
     public function show(Poll $poll)
     {
-        //
+        return new PollResource($poll);
```

Comment:
You should check if the auth user should have access to this resource

Reply:
Fixed in bfbd361ab18cd88b0c1f857054a57b444ffe19ce